src/App.tsx

```diff
@@ -64,7 +58,7 @@ function App(): JSX.Element {
   }
 
   // to get from px -> dp since we draw in the camera coordinate system
-  const SCALE = (format?.videoWidth ?? VIEW_WIDTH) / VIEW_WIDTH;
```

Comment:
Please don't hardcode this. This is not a good idea.

Above you removed the code that selected a Format, let's keep using a Format so we know video size in advance.